games-action/openclonk-7.0

```diff
@@ -0,0 +1,128 @@
+SUMMARY="Free multiplayer action game"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you
+control\ clonks, small, but witty and nimble humanoid beings. The game
+is mainly about\
+mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+        openclonk$secondaryArchSuffix = $portVersion
+        app:OpenClonk$secondaryArchSuffix = $portVersion
+	appdata$secondaryArchSuffix = $portVersion
+    	appdata(openclonk.appdata.xml)$secondaryArchSuffix = $portVersion
+    	application$secondaryArchSuffix = $portVersion
+    	application$secondaryArchSuffix = $portVersion
+    	bundled(timsort$secondaryArchSuffix = $portVersion
+   	mimehandler(x-scheme-handler/clonk)$secondaryArchSuffix = $portVersion
+   	openclonk(x86-64)$secondaryArchSuffix = $portVersion
+        "
+REQUIRES="
+        haiku$secondaryArchSuffix
+        lib:zlib$secondaryArchSuffix
+        lib:libpng$secondaryArchSuffix
+        lib:libjpeg_turbo$secondaryArchSuffix
+        lib:libfreetype$secondaryArchSuffix
+        lib:libogg$secondaryArchSuffix
+        lib:libvorbis$secondaryArchSuffix
+        lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
```

Comment:
I am almost absolutely sure you don't need X11 for this game.

Reply:
Yes, I used a linux dependency list as an example. I am new to this, just started developing.

Reply:
You are probably editing the recipe with StyleEdit, it's better to do it with Pe, you'll see the recipe structure better that way :)